cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -68,12 +71,28 @@ sub execute {
         $dbh->do("REPLACE INTO email_aliases VALUES (?, ?)", undef, $alias, $value);
         return $self->error("Database error: " . $dbh->errstr)
             if $dbh->err;
+        if ( $alias ne $alias_with_dashes ) {
+            $dbh->do( "REPLACE INTO email_aliases VALUES (?, ?)",
```

Comment:
I'm actually reconsidering this approach (sorry!) and thinking of going back to your original approach of modifying the select for the postfix.

The main problem with that is testing,  but it also means that we don't have to keep multiple things updated.